ci/docs/build.sh

```diff
@@ -18,27 +18,31 @@ export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
 export PROJECTS=(cuspatial)
 
-logger "Check environment..."
+gpuci_logger "Check environment"
 env
 
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage"
 nvidia-smi
 
-logger "Activate conda env..."
-source activate rapids
-# TODO: Move installs to docs-build-env meta package
-conda install -c anaconda markdown beautifulsoup4 jq
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
 pip install sphinx-markdown-tables
```

Comment:
```suggestion
```

This can be removed. It's included here: https://github.com/rapidsai/integration/blob/branch-0.17/conda/recipes/rapids-doc-env/meta.yaml#L49